integration-testing/examples/mpp-sample/build.gradle.kts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2016-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+buildscript {
+    val atomicfu_version = rootProject.properties["atomicfu_version"]
+
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath("org.jetbrains.kotlinx:atomicfu-gradle-plugin:$atomicfu_version")
+    }
+}
+
+group = "kotlinx.atomicfu.examples"
+version = "DUMMY_VERSION"
+
+plugins {
+    kotlin("multiplatform") version "${project.properties["kotlin_version"]}"
+    `maven-publish`
+}
+
+apply(plugin = "kotlinx-atomicfu")
+
+repositories {
+    mavenLocal()
+    maven("https://maven.pkg.jetbrains.space/kotlin/p/kotlin/dev")
+    mavenCentral()
+}
+
+kotlin {
+    jvm()
+    js()
+
+    val hostOs = System.getProperty("os.name")
+    val isArm64 = System.getProperty("os.arch") == "aarch64"
+    val isMingwX64 = hostOs.startsWith("Windows")
+    val nativeTarget = when {
```

Comment:
Does it make sense to enable all native targets (once it'll be possible to check them)?

Reply:
@mvicsokolova, is this done so that the names of the native target are always the same?

Reply:
That is just part of the default IDEA mpp project configuration. I guess, we need this to be able to call gradle build for this sample project on different targets on TC, right?

Reply:
All native targets could be enabled simultaneously and only those supported by the host compiler will be built.

Reply:
Added all the native targets